test/models/work_test.rb

```diff
@@ -0,0 +1,100 @@
+require "test_helper"
+
+describe Work do
```

Comment:
It might make sense to have some additional basic tests around the relationship with `Vote` -- you are relying on this relationship a lot in the custom model methods and so it would be good to have some separate tests that verify this independently.